na-toolbar.html

```diff
@@ -49,8 +49,12 @@
     </style>
     <iron-icon icon="bookmark"></iron-icon>
     <h1>Notes</h1>
-    <iron-icon icon="cloud" hidden$="[[!online]]"></iron-icon>
-    <iron-icon icon="cloud-off" hidden$="[[online]]"></iron-icon>
+    <span hidden$="[[!online]]">
+      <iron-icon icon="cloud"></iron-icon>
+    </span>
+    <span hidden$="[[online]]">
+      <iron-icon icon="cloud-off"></iron-icon>
+    </span>
```

Comment:
no, i mean in `iron-icon.html`, add a `:host([hidden])` style that makes `<iron-icon hidden>` work. We've added similar things in a bunch of elements after people complained `hidden` didn't work, because `hidden` is actually a lie